src/js/view/view.js

```diff
@@ -805,6 +814,8 @@ define([
             _showing = true;
             utils.removeClass(_playerElement, 'jw-flag-user-inactive');
 
+            _controlbar.checkCompactMode(_videoLayer.clientWidth);
+
```

Comment:
I'm not seeing any problems with it thrashing performance or causing a lot of repaints.  I'll check to see if it has issues in other browsers as well.  It's being called here to make sure that it's called at least once when the UI goes from being not displayed to being visible.  It could be run just once after if we call that only when `_showing` was false when we run `_userActivity`.